web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -108,6 +108,26 @@ public function viewModeList($content_type) {
     return $viewModes;
   }
 
+  /**
+   * Returns an array of taxonomy tags.
+   */
+  public function tagsList() {
```

Comment:
I don't think this method is in use. Should we delete it? or do you have something planned for it?

Reply:
Good catch - before I was using the `entity_autocomplete`, I was going to write my own. Removed.